README.md

```diff
@@ -68,12 +69,12 @@ NOTE: This example is superseded by the [NioCommand](https://github.com/nio-bloc
 
 HTTPRequestsPostSignal
 ======================
-Similar to the HTTPRequests block.  One request is made for every signal input.  The input signal will be used as the body of the post request.
+The HTTPRequestsPostSignal block is similar to the HTTPRequests block.  One request is made for every signal input.  The input signal will be used as the body of the post request.
 
 Properties
 ----------
 - **basic_auth_creds**: When making a request that needs Basic Authentication, enter the username and password.
-- **enrich**: If true, the output signal will include the original signal sent into the block
+- **enrich**: If false (unchecked), the output signal will include the original signal sent into the block
```

Comment:
Another idea: 
If checked (true), the original signal sent into the block will be excluded from the signal sent out of the block. If unchecked (false), the output signal will include the original signal sent into the block.

Reply:
What reads better? 
signal sent into the block | input signal | input sent into the block | input signal sent into the block
signal sent out from the block | output signal | output sent out by the block | output signal sent out by the block

Reply:
I like 'input signal' and 'output signal'.  I feel that should be sufficient for someone reading the docs.

Reply:
However in this context, I think the more explicit 'original signal sent into the block' is needed to explain the property